modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/content/Language_fr.properties

```diff
@@ -50,7 +50,7 @@ grant-type-x-is-unsupported-for-this-client-type=Le type d'autorisation «{0}»
 home-page-url=URL du site
 home-page-url-help=Fournir l'URL de la page d'accueil de votre application.
 home-page-url-is-invalid=L'URL du site Web n'est pas valide.
-home-page-url-scheme-is-invalid=La construction de l'URL de site Web n'est pas valide ; seules les constructions en http et https sont disponibles.
+home-page-url-scheme-is-invalid=La construction de l'URL de site Web n'est pas valide ; seules les constructions en http et https sont disponibles.
```

Comment:
Unsure if `0xa0` (non-breaking space) is critical here, to avoid breaking lines.